docker/resources/config-concord1/concord.config

```diff
@@ -1,162 +0,0 @@
-FEATURE_time_service: false
```

Comment:
Sorry in advance if this is a dumb question or I previously missed something important or I am overlooking something important, but can you comment on why this PR deletes the `docker/resources/config-concord*/concord.config` files? Was this intentional?  Looking at the other changes, it is not immediately evident to me how the deletion of these files is related to the addition of calls to the tutorial, and I do not see anything in the PR description clarifying how and/or why these deletions are part of this PR.